lib/java/gradle.properties

```diff
@@ -28,7 +28,9 @@ maven-repository-id=apache.releases.https
 httpclient.version=4.5.6
 httpcore.version=4.4.1
 slf4j.version=1.7.25
-servlet.version=2.5
+#servlet.version=2.5
+#It contains servlet3
+tomcat.embed.version=8.5.5
```

Comment:
Also, is there a reason not to use tomcat 9.x?

Reply:
Apache Tomcat 8.5 is one of the most frequently used Tomcat versions, and we tend to choose the most popular and stable version.

Reply:
If we decide to pick 8.5, maybe it's better to pick the latest one (8.5.46).

Reply:
checked it and agree with you, down that.